content/en/real_user_monitoring/guide/devtools-tips.md

```diff
@@ -24,7 +24,7 @@ In the picture above, the `console.error` function is instrumented. Notice that
 
 ### Adding scripts to your browser ignore list to display the correct file and line number
 
-Most browsers allow developers to select scripts and add them to an ignore list. To display the correct file and line number, you can add the following scripts to your browser's ignore list: `datadog-rum-v*.js` and `datadog-logs-v*.js`.
+Most browsers allow developers to select scripts and add them to an ignore list. To display the correct file and line number, you can add the following scripts to your browser's ignore list: `datadog-rum-v*.js`, `datadog-logs-v*.js`, `datadog-rum.js` and `datadog-logs.js`.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ Maybe simplify like this:

`datadog-rum*.js` `datadog-logs*.js`